query-connector/src/app/query/components/header/header.tsx

```diff
@@ -67,6 +120,26 @@ export default function HeaderComponent() {
           description="It's not! Data inputted into the TEFCA Query Connector is not persisted or stored anywhere."
         ></Modal>
       )}
+
+      {showMenu && (
+        <div className={styles.menuDropdownContainer}>
+          <ul
+            id="dropdown-menu"
+            className={`usa-nav__submenu ${styles.menuDropdown}`}
+          >
+            <li className={`usa-nav__submenu-item`}>
+              <a className={styles.menuItem} href="#">
+                My queries
```

Comment:
do we want to comment this out for now? until we get further along with the custom query page creation work, this will be a tad misleading to users/demos;